collectors/o365/lib/o365_mgmnt/o365management.js

```diff
@@ -125,12 +126,42 @@ class O365Management extends msRestAzure.AzureServiceClient {
         }
         httpRequest.headers.set('Content-Type', 'application/json; charset=utf-8');
         httpRequest.body = null;
-        // Create handler
+        // Request Handler
         const handler = this.requestHandler(httpRequest);
         // Send Request
         return client.sendRequest(httpRequest).then(handler);
     }
     
+    getNextSubscriptionsContentPage(nextPageUrl, options) {
```

Comment:
I'm sure this function can be used for any paged content.
I suggest we call it `getNextPageUrl`.
BTW, can't we use `getContent()` function for requesting nextPagetUrl?

Reply:
It can, and that was the original aim. I'll rename it.

and you are correct. We could just use `getContent` to retrieve the next page. All it's doing right now is just getting the body at the end.